csharp_dotnetcore/3.Welcome-Users/Startup.cs

```diff
@@ -67,16 +67,12 @@ public void ConfigureServices(IServiceCollection services)
                 // add the Microsoft.Bot.Builder.Azure  Nuget package to your solution. That package is found at:
                 // https://www.nuget.org/packages/Microsoft.Bot.Builder.Azure/
                 // IStorage Store = new Microsoft.Bot.Builder.Azure.AzureBlobStorage("AzureBlobConnectionString", "containerName");
-
-                // var convoState = new ConversationState(dataStore);
-                // options.State.Add(new ConversationState(dataStore));
                 options.State.Add(new UserState(dataStore));
 
                 // Add State to BotStateSet Middleware (that require auto-save)
                 // The BotStateSet Middleware forces state storage to auto-save when the Bot is complete processing the message.
                 // Note: Developers may choose not to add all the State providers to this Middleware if save is not required.
```

Comment:
This comment is no longer accurate, due to the name change to AutoSaveStateMiddlware.